packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -0,0 +1,296 @@
+        poi_resubmit_for_vanuatu,
+        poi_verified_for_bvi_labuan_maltainvest,
+        poi_or_poa_not_submitted,
+        poi_poa_verified_for_bvi_labuan_maltainvest,
+        need_poa_resubmission,
+        poi_acknowledged_for_bvi_labuan_maltainvest,
+        poa_acknowledged,
+        poa_pending,
+    } = getAuthenticationStatusInfo(account_status);
+
+    React.useEffect(() => {
+        if (is_jurisdiction_modal_visible) {
+            updateAccountStatus();
+            setJurisdictionSelectedShortcode('');
+            if (!has_submitted_personal_details) {
+                let get_settings_response: GetSettings = {};
+                if (!account_settings) {
+                    WS.authorized.storage.getSettings().then((response: GetAccountSettingsResponse) => {
+                        get_settings_response = response.get_settings as GetSettings;
+                        setAccountSettings(response.get_settings as GetSettings);
+                    });
+                } else {
+                    get_settings_response = account_settings;
+                }
+                const { citizen, place_of_birth, tax_residence, tax_identification_number, account_opening_reason } =
+                    get_settings_response;
+                if (citizen && place_of_birth && tax_residence && tax_identification_number && account_opening_reason) {
+                    setHasSubmittedPersonalDetails(true);
+                }
+            }
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_jurisdiction_modal_visible]);
+
+    React.useEffect(() => {
+        if (jurisdiction_selected_shortcode) {
+            setChecked(false);
+        }
+    }, [jurisdiction_selected_shortcode, is_jurisdiction_modal_visible]);
+
+    const financial_available_accounts = trading_platform_available_accounts.filter(
+        available_account => available_account.market_type === 'financial'
+    );
+
+    const synthetic_available_accounts = trading_platform_available_accounts.filter(
+        available_account => available_account.market_type === 'gaming'
+    );
+
+    const modal_title = is_eu
+        ? localize('Jurisdiction for your DMT5 CFDs account')
+        : localize('Choose a jurisdiction for your DMT5 {{account_type}} account', {
+              account_type: account_type.type === 'synthetic' ? 'Synthetic' : 'Financial',
+          });
+
+    const is_svg_selected = jurisdiction_selected_shortcode && jurisdiction_selected_shortcode === 'svg';
+    const is_bvi_selected = jurisdiction_selected_shortcode && jurisdiction_selected_shortcode === 'bvi';
+    const is_vanuatu_selected = jurisdiction_selected_shortcode && jurisdiction_selected_shortcode === 'vanuatu';
+    const is_labuan_selected = jurisdiction_selected_shortcode && jurisdiction_selected_shortcode === 'labuan';
+    const is_maltainvest_selected =
+        jurisdiction_selected_shortcode && jurisdiction_selected_shortcode === 'maltainvest';
```

Comment:
```suggestion
    const is_svg_selected = jurisdiction_selected_shortcode === 'svg';
    const is_bvi_selected = jurisdiction_selected_shortcode === 'bvi';
    const is_vanuatu_selected = jurisdiction_selected_shortcode === 'vanuatu';
    const is_labuan_selected = jurisdiction_selected_shortcode === 'labuan';
    const is_maltainvest_selected = jurisdiction_selected_shortcode === 'maltainvest';
```
@amina-deriv `jurisdiction_selected_shortcode &&`  would make sense if one of them was like  `= jurisdiction_selected_shortcode !== 'svg'` 🙂